### PR TITLE
Add scope to documentation

### PR DESCRIPTION
**Pull request by nikstuckenbrock**
I'm sorry for the mess! I've somehow managed to destroy my previous PR #106 and have created this one to fix the history problems. Sorry for the chaos. :(

I would appreciate if you could add the HACKTOBERFEST-ACCEPTED label to the pull request.

### CODE REVIEW

File: docs/docs/b2c/fastapi_configuration.mdx [hunk 1]
```diff
@@ -350,27 +403,55 @@ views based on the scope.
...
+    
+    @computed_field
+    @property
+    def OPENID_CONFIG_URL(self) -> dict:
+        return f'https://{self.TENANT_NAME}.b2clogin.com/{self.TENANT_NAME}.onmicrosoft.com/{self.AUTH_POLICY_NAME}/v2.0/.well-known/openid-configuration'
+    
+    @computed_field
+    @property
+    def OPENAPI_AUTHORIZATION_URL(self) -> dict:
+        retutn f'https://{settings.TENANT_NAME}.b2clogin.com/{settings.TENANT_NAME}.onmicrosoft.com/{settings.AUTH_POLICY_NAME}/oauth2/v2.0/authorize'
```
Comment (davidhuser) [inline docs/docs/b2c/fastapi_configuration.mdx:443]: typo at `retutn`

**Comment by davidhuser** [comment]
just did a quick scan, I think it misses setting the default scope during FastAPI.app instantiation, see bottom part here https://github.com/Intility/fastapi-azure-auth/issues/106#issuecomment-1759125399

File: docs/docs/single-tenant/fastapi_configuration.mdx [hunk 2]
```diff
@@ -185,23 +185,46 @@ We've used two flags: `usePkceWithAuthorizationCodeGrant`, which is the authenti
 
 Now, the fun part begins! 🚀
 
-Import the `SingleTenantAzureAuthorizationCodeBearer` from `fastapi_azure_auth` and configure it:
+Import the `SingleTenantAzureAuthorizationCodeBearer` from `fastapi_azure_auth` and configure it. You can use the `Settings` class to compute the required URLs:
```
Comment (davidhuser) [inline docs/docs/single-tenant/fastapi_configuration.mdx:188]: nitpick but we also compute other properties, so maybe "... compute the required fields"

File: docs/docs/single-tenant/fastapi_configuration.mdx [hunk 3]
```diff
@@ -299,9 +347,9 @@ if settings.BACKEND_CORS_ORIGINS:
 azure_scheme = SingleTenantAzureAuthorizationCodeBearer(
     app_client_id=settings.APP_CLIENT_ID,
     tenant_id=settings.TENANT_ID,
-    scopes={
-        f'api://{settings.APP_CLIENT_ID}/user_impersonation': 'user_impersonation',
-    }
+    scopes=settings.SCOPES,
+    openapi_authorization_url=settings.OPENAPI_AUTHORIZATION_URL,
```
Comment (davidhuser) [inline docs/docs/single-tenant/fastapi_configuration.mdx:351]: These are new params added to [SingleTenant](https://intility.github.io/fastapi-azure-auth/single-tenant/fastapi_configuration) not previously in the docs. I don't have much experience with Single Tenant and would refer to @JonasKs

**Review by davidhuser — APPROVED** [review]
LGTM

**Comment by JonasKs** [comment]
Thank you so much 😊

**Pull request merged**